### PR TITLE
Modernize ui-component tests

### PR DESCRIPTION
**Pull request by blink1073**
## References
Continuation of #8124 for the `ui-components` package. 

## Code changes
Upgrades the test suite of `ui-components`.  Updated the import of the svg string.

## User-facing changes
Easier to contribute to this package.

## Backwards-incompatible changes
None

### CODE REVIEW

File: packages/ui-components/babel.config.js [hunk 1]
```diff
@@ -1,12 +1 @@
...
-      '@babel/preset-env',
-      {
-        targets: {
-          node: 'current'
-        }
-      }
-    ]
-  ]
-};
+module.exports = require('@jupyterlab/testutils/lib/babel.config');
```
Comment (telamonian) [inline packages/ui-components/babel.config.js:1]: This was here originally to support the storybook stuff, right? Are there any potential issues in having the storybook/test babel.config overlap like this?

File: packages/ui-components/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "attach",
+            "name": "Attach",
```
Comment (telamonian) [inline packages/ui-components/.vscode/launch.json:7]: Can we give this debug config a more descriptive name, maybe "Attach to Jest test"? vscode allows comments in these json files, so we may also want to include detailed inline instructions on exactly how to spin this debug job up. I assume it's:

- `jlpm test:debug:watch`
- run this debug job in vscode

File: packages/ui-components/babel.config.js (hunk 1 above)
Comment (blink1073) [reply packages/ui-components/babel.config.js:1]: Good catch, thanks!  I'll make sure storybook is working before we consider merging this.

File: packages/ui-components/.vscode/launch.json (hunk 2 above)
Comment (blink1073) [reply packages/ui-components/.vscode/launch.json:7]: I like it, but as a single sweep after the migration is complete. Mind opening an issue?

Comment (telamonian) [reply packages/ui-components/.vscode/launch.json:7]: done: #8327

File: packages/ui-components/babel.config.js (hunk 1 above)
Comment (telamonian) [reply packages/ui-components/babel.config.js:1]: I think the contents of testutils/lib/babel.config and ui-components/babel.config.js are currently identical, so it shouldn't cause an issue right now. What I was wondering is if there was any reason for them to diverge in the future. If not, then it should be fine

Comment (blink1073) [reply packages/ui-components/babel.config.js:1]: I added a CI test to make sure storybook builds.

File: packages/ui-components/.vscode/launch.json (hunk 2 above)
Comment (blink1073) [reply packages/ui-components/.vscode/launch.json:7]: Thanks!

**Pull request merged**